test/rules/noUnusedVariableRuleTests.ts

```diff
@@ -113,4 +113,70 @@ describe("<no-unused-variable>", () => {
 
         assert.lengthOf(actualFailures, 0);
     });
+
+    describe("react option set to", () => {
```

Comment:
Good points; actually forgot to refactor the test cases given the change in option. Will do